src/Audio/Cue.cs

```diff
@@ -774,6 +783,14 @@ internal bool INTERNAL_update()
 								long elapsedFromStop = INTERNAL_timer.ElapsedMilliseconds - INTERNAL_fadeStart;
 								variableValue = elapsedFromStop;
 							}
+							else
+							{
+								variableValue = 0.0f;
+							}
+						}
+						else if (curRPC.Variable.Equals("NumCueInstances"))
```

Comment:
Nevermind, it was a bug in FNA!

https://github.com/FNA-XNA/FNA/commit/34ec74f48c286203aa27dc7c5a55b86e2eb8878d